src/include/pbs_nodes.h

```diff
@@ -265,7 +266,7 @@ struct	pbsnode {
 	long			 nd_nsnfree;	/* number of VPs free */
 	long			 nd_ncpus;	/* number of phy cpus on node */
 	short			 nd_written;	/* written to nodes file */
-	unsigned short		 nd_state;	/* state of node */
+	unsigned long		 nd_state;	/* state of node */
```

Comment:
Thanks for looking this up.  I'll change it to an int.  I think everything needs to be changed at once.  IMHO, storing a long in the code and an int in the DB is just a timebomb waiting to happen.